### PR TITLE
Updating doc guidelines for not using source,bash

### PR DESCRIPTION
**Pull request by bergerhoffer**
@openshift/team-documentation For peer review.

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@bergerhoffer this would apply to all content, not just 4.0, right?

**Comment by bergerhoffer** [comment]
@vikram-redhat Yes probably, though I only went through and remove source,bash instances from the 4.0 docs.

As for the guidance, I thought that enterprise-4.0 would be replacing master, so that would eventually becoming guidance for all? Or do you want me to submit an update to the master version of the guidance?

**Comment by vikram-redhat** [comment]
Hey @bergerhoffer - yes that is right. enterprise-4.0 will become master, but we will still have guidance for 3.x docs (when 4.0 becomes master, we will move the current master guidance to 3.11, the last 3.x release.)

**Comment by vikram-redhat** [comment]
FWIW, I am happy for us to not remove the source, bash from 3.x docs at this stage, just have it in the guidance for any future work.

**Comment by bergerhoffer** [comment]
@vikram-redhat I opened https://github.com/openshift/openshift-docs/pull/13977 to update the guidance on master for 3.x, if you want to take a look.

File: contributing_to_docs/doc_guidelines.adoc [hunk 1]
```diff
@@ -452,12 +452,13 @@ $ lorem.sh
 Without the line spaces, the content is likely to be not parsed correctly.
 
 * It is recommended to include source tags for the programming language used in the code block to enable syntax highlighting. For example, use the source tags
- `[source, bash]`, `[source, yaml]`, `[source, javascript]`.
+ `[source, yaml]` or `[source, javascript]`.
++
+NOTE: Do not use `[source, bash]` for `oc` commands or any terminal commands, as it does not render properly in some cases, so we leave off for consistency across the docs.
```
Comment (kalexand-rh) [inline contributing_to_docs/doc_guidelines.adoc]: s/as/because
s/cases, so we leave off for consistency across the docs./cases.

**Review by kalexand-rh — COMMENTED** [review]
I missed that you had PRs for both streams.

**Pull request merged**